trax/data/__init__.py

```diff
@@ -43,6 +44,8 @@ def data_configure(*args, **kwargs):
 Shuffle = data_configure(inputs.Shuffle)
 shuffle = inputs.shuffle
 TFDS = data_configure(tf_inputs.TFDS)
+bert_tokenizer = data_configure(bert_inputs.bert_tokenizer)
+bert_glue_inputs = data_configure(bert_inputs.bert_glue_inputs)
```

Comment:
This will need to become a new function in `tf_inputs`.

Reply:
They both are removed, as they are unnecessary after simplifying pipeline and using TFDS building blocks and make_inputs function